static/form.js

```diff
@@ -1,29 +1,30 @@
-
 $(document).ready(function () {
-  $('.datepicker').datepicker({
-      format: 'mm-dd-yyyy',
-      autoclose: true,
-      todayHighlight: true,
-  });
+    $('.datepicker').datepicker({
+        format: 'mm-dd-yyyy',
```

Comment:
if you could change this to yyyy-mm-dd with the time as well for that fix we talked about.